qcfractal/queue/managers.py

```diff
@@ -592,7 +596,9 @@ def test(self, n=1) -> bool:
                         failed_program = program
                         break
                 if failed_program not in fail_report:
-                    fail_report[failed_program] = f"On test {k}: {result.error}"
+                    fail_report[failed_program] = f"On test {k}:" \
+                                                  f"\nException Type: {result.error.error_type}" \
+                                                  f"\nException Message: {result.error.error_message}"
```

Comment:
I did not see this here, I think I'll remove this in my own PR since this should be better

Reply:
I need to rebase anyway, wouldn't worry about it.